samples/tacos/src/main/res/values-night/themes.xml

```diff
@@ -0,0 +1,31 @@
+
+<resources>
+    <style name="TacoTheme" parent="Theme.Material3.Dark.NoActionBar">
+        <item name="colorPrimary">@color/md_theme_dark_primary</item>
+        <item name="colorOnPrimary">@color/md_theme_dark_onPrimary</item>
```

Comment:
These xml colors and themes aren't needed right?

Reply:
yeah I think since it's all compose we shouldn't need to do much in the xml layer 🤞